script/moveServiceWorker.js

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs'
```

Comment:
Since it's a runnable script that node will run I felt this was fine.

Reply:
Astro didn't play nice, it really wanted to compile the file afterwards when we run `pnpm run dev` and `ts-node` was angry. 
I feel like small scripts like this can benefit from being only `.js`